From: =?utf-8?q?J=C3=A9r=C3=A9my_Bobbio?= <lunar@debian.org>
Date: Sat, 12 Sep 2015 20:54:22 +0200
Subject: Use IO#each_line instead of IO#lines in poparser

IO#lines is deprecated and gives us a quite obnoxious deprecation warning.
---
 lib/fast_gettext/vendor/poparser.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/fast_gettext/vendor/poparser.rb b/lib/fast_gettext/vendor/poparser.rb
index 0b18880..27de150 100644
--- a/lib/fast_gettext/vendor/poparser.rb
+++ b/lib/fast_gettext/vendor/poparser.rb
@@ -144,7 +144,7 @@ module GetText
 
   def detect_file_encoding(po_file)
     open(po_file, :encoding => 'ASCII-8BIT') do |input|
-      input.lines.each do |line|
+      input.each_line.each do |line|
         return Encoding.find($1) if %r["Content-Type:.*\scharset=(.*)\\n"] =~ line
       end
     end
